flink-libraries/flink-gelly-examples/src/test/java/org/apache/flink/graph/library/TriangleEnumeratorITCase.java

```diff
@@ -47,10 +47,11 @@ public void testTriangleEnumerator() throws Exception	{
 				env);
 
 		List<Tuple3<Long,Long,Long>> actualOutput = graph.run(new TriangleEnumerator<Long, NullValue, NullValue>()).collect();
-		List<Tuple3<Long,Long,Long>>  expectedResult = TriangleCountData.getListOfTriangles();
+		List<Tuple3<Long,Long,Long>> expectedResult = TriangleCountData.getListOfTriangles();
 
-		Assert.assertEquals(actualOutput.size(), expectedResult.size());
+		Assert.assertEquals(expectedResult.size(), actualOutput.size());
 		for(Tuple3<Long,Long,Long> resultTriangle:actualOutput)	{
+			System.out.println(resultTriangle);
```

Comment:
Do we need to print the result? We try to avoid printing from tests unless it's absolutely necessary.